docs/fsharp-cheatsheet.md

```diff
@@ -6,30 +6,30 @@ The Microsoft [F# Documentation](https://learn.microsoft.com/en-us/dotnet/fsharp
 
 Contents
 --------
-[Comments](#Comments)  
-[Strings](#Strings)  
-[Basic Types and Literals](#BasicTypesAndLiterals)  
-[Functions](#Functions)  
-[Pattern Matching](#PatternMatching)  
-[Collections](#Collections)  
-[Tuples and Records](#TuplesAndRecords)  
-[Discriminated Unions](#DiscriminatedUnions)  
-[Exceptions](#Exceptions)  
-[Classes and Inheritance](#ClassesAndInheritance)  
-[Interfaces and Object Expressions](#InterfacesAndObjectExpressions)  
-[Active Patterns](#ActivePatterns)  
+[Comments](#Comments)
+[Strings](#Strings)
+[Basic Types and Literals](#BasicTypesAndLiterals)
+[Functions](#Functions)
+[Pattern Matching](#PatternMatching)
+[Collections](#Collections)
+[Tuples and Records](#TuplesAndRecords)
+[Discriminated Unions](#DiscriminatedUnions)
+[Exceptions](#Exceptions)
+[Classes and Inheritance](#ClassesAndInheritance)
+[Interfaces and Object Expressions](#InterfacesAndObjectExpressions)
+[Active Patterns](#ActivePatterns)
 [Compiler Directives](#CompilerDirectives)
 
 <a name="Comments"></a>Comments
 --------
 Block comments are placed between `(*` and `*)`. Line comments start from `//` and continue until the end of the line.
 
-	(* This is block comment *)
+    (* This is block comment *)
 
     // And this is line comment
-    
-XML doc comments come after `///` allowing us to use XML tags to generate documentation.    
-    
+
+XML doc comments come after `///` allowing us to use XML tags to generate documentation.
```

Comment:
should this explain that backticks dont mean anything and that you need a `<summary>` if you want `<c>` to do their job ;)